RunningOrder/Story/Views/SortMenu.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SortMenu.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 13/07/2021.
+//  Copyright © 2021 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct SortMenu: View {
+    @Binding var selectedSort: Option
+
+    func newIsReversed(for optionType: Option.OptionType) -> Bool {
```

Comment:
weird func name

Reply:
yes ! besides I don't need this function anymore as it is used only at one place, and only one small expression :) changed !